cmd/frontend/backend/external_services.go

```diff
@@ -72,13 +72,13 @@ func (e *externalServices) SyncExternalService(ctx context.Context, svc *types.E
 	return err
 }
 
-// ExcludeRepoFromExternalService excludes given repo from given external service config.
+// ExcludeRepoFromExternalServices excludes given repo from given external services configs.
 //
 // Function is pretty beefy, what it does is:
-// - finds an external service by ID and checks if it supports repo exclusion
+// - finds external services by IDs and checks if they support repo exclusion
 // - adds repo to `exclude` config parameter and updates an external service
-// - triggers external service sync
-func (e *externalServices) ExcludeRepoFromExternalService(ctx context.Context, externalServiceID int64, repoID api.RepoID) error {
+// - triggers external services syncs
```

Comment:
Nit: The current form is already right if you use `syncs` here.

```suggestion
// - triggers external service syncs
```